js/script.js

```diff
@@ -91,6 +91,7 @@ var script = {
 
 	"h Go ahead and create an amazing Game!",
 	"h I can't wait to see what story you'll tell!",
+	"pee in my mouth",
 	"end"
```

Comment:
You should really not add this into a commit you do. It'll be there forever in the commit history if I were to merge it in.